packages/runtime/container-runtime/src/opLifecycle/opSplitter.ts

```diff
@@ -215,6 +216,7 @@ export class OpSplitter {
 		// back-compat with 1.x builds
 		// This is only required / present for non-compressed, chunked ops
 		// For compressed ops, we have op grouping enabled, and type of each op is preserved within compressed content.
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
```

Comment:
Seeing this `originalType` used in two places but requiring the cast makes me very suspicious of the typing here 🤔 .

Reply:
Yeah. This was odd enough that I erred on the side of leaving it alone.